src/services/panthalassa/index.js

```diff
@@ -0,0 +1,318 @@
+}
+
+/**
+ * @desc Converts Account's Ethereum public key to an address
+ * @param {string} publicKey User's public key
+ * @returns {Promise<*>} {string} Account's Ethereum public key
+ */
+export async function panthalassaEthPubToAddress(publicKey: String):Promise<string> {
+  return Panthalassa.PanthalassaEthPubToAddress(publicKey);
+}
+
+/**
+ * @desc Retrieves Account's Ethereum address
+ * @returns {Promise<*>} {string} An address
+ */
+export async function panthalassaEthAddress():Promise<string> {
+  return Panthalassa.PanthalassaEthAddress();
+}
+
+/**
+ * @desc Retrieves Account's public ey
+ * @returns {Promise<*>} {string} Account's public key
+ */
+export async function panthalassaGetIdentityPublicKey():Promise<string> {
+  return Panthalassa.PanthalassaGetIdentityPublicKey();
+}
+
+/**
+ * @desc Creates Account's signed profile
+ * @param {string} name Profile's name to show
+ * @param {string} location Profile's location
+ * @param {string} image Profile's avatar
+ * @param {string} keyManagerStore Account's ConfigurationStore
+ * @param {string} password Account's password
+ * @returns {Promise<*>} {string} Account's signed profile
+ */
+export async function panthalassaSignProfileStandAlone(name: String, location: String, image: String, keyManagerStore: String, password: String):Promise<string> {
+  return Panthalassa.PanthalassaSignProfileStandAlone({
+    name,
+    location,
+    image,
+    keyManagerStore,
+    password,
+  });
+}
+
+/**
+ * @desc Updates information to the Account's profile
+ * @param {string} name Profile's name to show
+ * @param {string} location Profile's location
+ * @param {string} image Profile's avatar
+ * @returns {Promise<*>} {
+ */
+export async function panthalassaSignProfile(name: String, location: String, image: String):Promise<string> {
+  return Panthalassa.PanthalassaSignProfile({
+    name,
+    location,
+    image,
+  });
+}
+
+/**
+ * @desc Retrieves Account's mnemonic phrase
+ * @returns {Promise<*>} {string} Mnemonic phrase
+ */
+export async function panthalassaGetMnemonic():Promise<string> {
+  return Panthalassa.PanthalassaGetMnemonic();
+}
+
+/**
+ * @desc Starts DApp's VM
+ * @param {string} dApp DApp's Identifier
+ * @param {string} timeout Value for timeout operation
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaStartDApp(dApp: String, timeout: Number):Promise<boolean> {
+  return Panthalassa.PanthalassaStartDApp({
+    dApp,
+    timeout,
+  });
+}
+
+/**
+ * @desc Opens an specific Dapp
+ * @param {string} id DApp's Id
+ * @param {string} context TODO
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaOpenDApp(id: String, context: String):Promise<boolean> {
+  return Panthalassa.PanthalassaOpenDApp({
+    id,
+    context,
+  });
+}
+
+/**
+ * @desc TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaNewPreKeyBundle():Promise<string> {
```

Comment:
I don't think it's needed, since chat is moved to Panthalassa

Reply:
Deleted